Hakawai/Mentions/HKWMentionsPlugin.m

```diff
@@ -1266,6 +1268,47 @@ - (BOOL)advanceStateForStringDeletionAtRange:(NSRange)range
     return YES;
 }
 
+/*!
+ Advance the state machine when multiple characters are deleted with the Korean Mentions Fix on
+ */
+- (BOOL)korean_mentions_fix_advanceStateForStringDeletionAtRange:(NSRange)range
```

Comment:
this is almost exactly the same as advanceStateForStringDeletionAtRange, just without the